src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -265,6 +264,19 @@ protected function getAllowedAttributes($classOrObject, array $context, bool $at
         return $allowedAttributes;
     }
 
+    /**
+     * @return array|false False on no group specified
+     *
+     * @internal
```

Comment:
does it need to be internal?

Reply:
No, but I feel it would not be the scope of this PR to open such a method to userland?

Reply:
I think we should expose it since it's useful.